test/Sprites/Color.test.cpp

```diff
@@ -0,0 +1,67 @@
+#include "../src/Sprites/Color.h"
+#include <gtest/gtest.h>
+
+TEST(Color, CompareEqual) {
+	// Note: Internally EXPECT_EQ uses `operator ==`
+	//       EXPECT_EQ has better error formatting than calling == explicitly
+
```

Comment:
Thank you for adding this comment. I was wondering why the code was structured how it was before seeing it.